iOS_SDK/OneSignalSDK/Source/UIApplicationDelegate+OneSignal.m

```diff
@@ -75,7 +76,7 @@ - (void) setOneSignalDelegate:(id<UIApplicationDelegate>)delegate {
     
     Class delegateClass = [delegate class];
     
-    if (delegate == nil || [swizzledClasses containsObject:delegateClass]) {
+    if (delegate == nil || [OneSignalAppDelegate swizzledClassInHeirarchy:delegateClass]) {
```

Comment:
Can we make this same logic change to `UNUserNotificationCenter+OneSignal.m`? It possible this same problem could happen with that delegate and think we should keep the implementations the same.

Reply:
Done!